lldb/source/Utility/ProcessInfo.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include <climits>
 #include <optional>
+#include <sys/resource.h>
```

Comment:
Not every OS will have this header file, so I am not sure we can/should be importing this header. Windows build will most likely fail and any non unix based OS as well.